CHANGELOG.md

```diff
@@ -42,17 +42,11 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
   describe.skip("test", () => {});
   it.skip("test", () => {});
   ```
-<<<<<<< HEAD
-=======
 
-- Add the rule [noFocusedTests](https://biomejs.dev/linter/rules/no-focused-tests), to disallow skipped tests:
+#### Bug fixes
 
-  ```js
-  describe.only("test", () => {});
-  it.only("test", () => {});
-  ```
+- Fix [#1640](https://github.com/biomejs/biome/issues/1640). [useEnumInitializers](https://biomejs.dev/linter/rules/use-enum-initializers) code action now generates valid code when last member has a comment but no comma. Contributed by @kalleep
 
->>>>>>> fd3de977d1 (feat(linter): new rule noFocusedTests (#1641))
```

Comment:
Seems like this was commited by mistake https://github.com/biomejs/biome/pull/1635/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR45-R55